besu/src/test/java/org/hyperledger/besu/controller/TransitionControllerBuilderTest.java

```diff
@@ -76,6 +77,7 @@ public void setup() {
             new TransitionProtocolSchedule(
                 preMergeProtocolSchedule, postMergeProtocolSchedule, mergeContext));
     cliqueBuilder.nodeKey(NodeKeyUtils.generate());
+    postMergeBuilder.storageProvider(new InMemoryKeyValueStorageProvider());
```

Comment:
powBuilder also needs storage provider here.  Also in buildTransitionCoordinator, it needs storage.  Then these tests should 👍